
This replicates on the t430s the patches that zmatt needed to do to get
his aftermarket battery to validate.

Tested and known to work.


@@ -7182,14 +7182,14 @@
 0001c0d0  04 10 fa 16 01 11 42 29  00 02 d8 1e 02 10 d9 1e  |......B)........|
 0001c0e0  42 10 1b ff 42 28 01 02  da 1e 42 10 db 1e 02 10  |B...B(....B.....|
 0001c0f0  c9 70 0a d9 3c da 04 db  18 f1 c4 16 01 10 c7 09  |.p..<...........|
-0001c100  5e 80 c7 16 81 10 8c e9  88 e0 ca 21 81 0f 00 00  |^..........!....|
+0001c100  5e 80 c7 16 81 10 e0 78  88 e0 ca 21 81 0f 00 00  |^......x...!....|
 0001c110  64 00 fc 0e 21 f2 ca 20  c1 03 0d f0 05 8e 01 e0  |d...!.. ........|
 0001c120  0f 78 88 e0 4e 02 2a 00  05 ae 07 d8 c5 03 20 00  |.x..N.*....... .|
 0001c130  04 ae 7f 02 20 00 04 8e  c9 70 0b d9 eb 05 ef ff  |.... ....p......|
 0001c140  3c da c5 16 82 10 55 26  43 1c 59 63 58 63 32 89  |<.....U&C.YcXc2.|
 0001c150  10 88 00 23 90 00 08 b9  00 21 04 00 58 63 13 88  |...#.....!..Xc..|
 0001c160  59 63 31 89 2f 24 08 01  08 b8 38 60 10 78 88 71  |Yc1./$....8`.x.q|
-0001c170  e6 0f 2f f3 fa 16 02 11  0b 08 51 00 00 86 83 b8  |../.......Q.....|
+0001c170  e6 0f 2f f3 fa 16 02 11  4a 26 00 70 00 86 83 b8  |../.....J&.p....|
 0001c180  24 f0 10 10 80 20 11 10  81 20 08 b8 38 60 12 10  |$.... ... ..8`..|
 0001c190  81 20 10 78 13 10 82 20  08 b9 59 61 30 79 f2 0f  |. .x... ..Ya0y..|
 0001c1a0  2f f3 fa 16 02 11 0f 08  51 00 00 86 45 20 00 0a  |/.......Q...E ..|
@@ -7205,13 +7205,13 @@
 0001c240  04 f4 13 00 20 00 b2 d1  b0 d0 21 80 14 6a 58 60  |.... .....!..jX`|
 0001c250  14 79 4a 24 40 74 56 26  c2 16 a8 20 c0 01 01 11  |.yJ$@tV&... ....|
 0001c260  80 04 01 1a 12 00 c9 70  0a d9 27 da bd 04 ef ff  |.......p..'.....|
-0001c270  11 db c4 16 00 10 e1 b8  25 f3 c7 16 80 10 88 e8  |........%.......|
+0001c270  11 db c4 16 00 10 e1 b8  25 f3 c7 16 80 10 e0 78  |........%......x|
 0001c280  e9 70 8e 0d 2f f2 8a 21  0f 0a 55 f1 05 8e 01 e0  |.p../..!..U.....|
 0001c290  0f 78 69 08 33 02 05 ae  2e f0 d2 0e 2f f2 e9 70  |.xi.3......./..p|
 0001c2a0  81 e0 11 f5 47 f1 c9 70  0b d9 7f 04 ef ff 28 da  |....G..p......(.|
 0001c2b0  08 16 c2 10 8c 22 ff 8f  04 f4 17 00 20 00 95 d1  |....."...... ...|
 0001c2c0  92 d0 34 6a 59 61 3b 79  01 80 11 e0 19 61 26 0f  |..4jYa;y.....a&.|
-0001c2d0  ef fe 56 26 c0 16 89 e8  00 86 a4 b8 00 a6 14 d8  |..V&............|
+0001c2d0  ef fe 56 26 c0 16 e0 78  00 86 a4 b8 00 a6 14 d8  |..V&...x........|
 0001c2e0  0f 02 20 00 04 ae 06 16  c0 10 61 b8 0d 78 0d 08  |.. .......a..x..|
 0001c2f0  72 00 06 ae 0e d8 f6 f1  00 86 84 b8 f0 f1 a5 ae  |r...............|
 0001c300  05 d8 04 ae c9 70 a5 fd  e9 01 00 00 21 96 15 d8  |.....p......!...|
